src/components/Header/index.jsx

```diff
@@ -25,11 +27,21 @@ class Header extends React.Component {
             <div className="col-md-auto">
               <Link to="/">Courses</Link>
             </div>
+            <Dropdown
```

Comment:
Maybe add some bootstrap classes to make this float-right. It looks too weird on the left, and we know we're going to need to move it right eventually. Might as well do it now.